README.md

```diff
@@ -271,7 +271,7 @@ infos := &openapi.Info{
    Description: `This is a sample Fruits market server.`,
    Version:     "1.0.0",
 }
-f.GET("/openapi.json", nil, fizz.OpenAPI(infos, "json"))
+f.GET("/openapi.json", nil, f.OpenAPI(infos, "json"))
```

Comment:
`f` is often used in the examples to represent a `fizz.Fizz` instance. For clarity, I think we should add a declaration in this example such as `f := fizz.New()` using [`fizz.New`](https://pkg.go.dev/github.com/wI2L/fizz#New).

Reply:
That makes sense. Added.

Reply:
Thanks.